app/src/test/java/gov/va/vro/controller/VroControllerTest.java

```diff
@@ -122,9 +120,8 @@ void postFullHealthAssessment() throws Exception {
     assertEquals(request.getDiagnosticCode(), response2.getDiagnosticCode());
     assertEquals(request.getVeteranIcn(), response2.getVeteranIcn());
 
-    Optional<ClaimEntity> claimEntityOptional =
-        claimRepository.findByClaimSubmissionIdAndIdType("1234", "va.gov-Form526Submission");
-    assertTrue(claimEntityOptional.isPresent());
+    var claim = claimRepository.findByVbmsId(request.getClaimSubmissionId());
```

Comment:
For MAS requests, the claimSubmissionId (referenceId) is the collectionId.
For claims, the vbmsID is the benefitsClaimId, as provided by MAS.
Thus, the vbmsID is not comparable to the claimSubmissionId.

See https://dsva.slack.com/archives/C01Q7979Z7D/p1674145386628059?thread_ts=1674144499.310049&cid=C01Q7979Z7D and https://dsva.slack.com/archives/C01Q7979Z7D/p1674237329439659?thread_ts=1674144499.310049&cid=C01Q7979Z7D

Reply:
Resolved by larger code commits later in the code.